gatsby/content/blog/2020/04/2020-04-06-selfhosting-matrix.mdx

```diff
@@ -0,0 +1,162 @@
+improving Synapse's performance) - so it's particularly important right now to
+help folks run their own servers.
+
+Therefore we're very happy to announce that it's easier than ever before now to
+self-host your own video conferencing alongside Riot & Synapse: as of Riot/Web
+1.5.15 (released last week), it's now a single config option to point Riot at
+a specific Jitsi rather than needing to hook up to an integration manager!
+
+Meanwhile, over the last 18 months, it's got easier and easier to run your
+own Matrix deployments: the Debian packages are unrecognisably better now, and
+with `.well-known` URL support it's trivial to set up federation without
+needing to worry about complicated DNS, TLS or load balancer configurations.
+
+So, to try to show off just how smooth this has become, we thought we'd do a
+run-through video showing installing Synapse, Riot & Jitsi on a completely
+fresh Debian install.  It's (almost) filmed in a single shot, and takes about
+20 minutes from beginning to end.
+
+Please note that this does assume you're pretty familiar with Linux system
+administration.  If you're not, then we'd recommend using a Matrix hosting
+provider such as [Modular.im](https://modular.im) (which directly supports development of the core team),
+[Ungleich.ch](https://ungleich.ch/u/products/hosted-matrix-chat/), or [StartupStack](https://startupstack.tech/).
+
+Finally, while the video shows how to install on Debian via Debian packages,
+there are many many other environments and architectures (e.g. installing
+under Docker) - this is just one relatively easy way to skin the cat.  Perhaps
+there will be other 'speed-run' videos in future :)
+
+<iframe width="1229" height="691" src="https://www.youtube.com/embed/dDddKmdLEdg" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If you want to follow along at home without listening to the video (and I can't blame you if you do ;) the high level steps are as follows:
+
+### Debian & DNS
+
+ * Take one fresh Debian 10 install.
+ * Point the DNS for your domain to it.  You should use separate subdomains for the various services as a hygiene measure to make cross-site scripting attacks less effective.  In this example, we set up DNS for:
+   * `dangerousdemos.net` (general website, and for hosting a .well-known path to advertise the Matrix service)
+   * `matrix.dangerousdemos.net` (Synapse)
+   * `riot.dangerousdemos.net` (Riot/Web)
+   * `jitsi.dangerousdemos.net` (Jitsi video conferencing)
+   * In practice, we used a `*.dangerousdemos.net` wildcard DNS record for the three subdomains in this instance.
+
+### Nginx and LetsEncrypt
+
+ * Install nginx as a webserver: `apt-get update && apt -y install nginx`
+ * Go to `/etc/nginx/sites-enabled` and copy the vhost configuration block from the bottom of `default` to new files called `dangerousdemos.net`, `matrix.dangerousdemos.net`, and `riot.dangerousdemos.net`.
```

Comment:
It feels a bit weird that there's no mention of the Jitsi domain here, maybe add something along the lines of "we don't handle `jitsi.dangerousdemos.net` here because that's a separate step"?